src/target.cpp

```diff
@@ -743,6 +743,22 @@ uint32_t target_c_type_size_in_bits(const ZigTarget *target, CIntType id) {
         case OsLv2:
         case OsNetBSD:
```

Comment:
looks like you need to shift these down so they get the panic():

```
         case OsAnanas:
         case OsCloudABI:
         case OsDragonFly:
         case OsFreeBSD:
         case OsIOS:
         case OsKFreeBSD:
         case OsLv2:
         case OsNetBSD: 
```

Reply:
Moved the targets.